commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -30,7 +30,7 @@
                     <h1 class="hosted__heading hosted-tone--renault">@page.video.title</h1>
                 </div>
                 <div class="hostedbadge hostedbadge--shouldscale">
-                    <img class="hostedbadge__logo" src="@page.logoUrl">
+                    <img class="hostedbadge__logo" src="@page.logoUrl" alt="hosted logo Renault">
```

Comment:
What would we want a screen reader to actually announce? I'd wager just "Renault" - or, if the logo is merely decorative, nothing at all.